zendev/cmd/test.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import subprocess
+import sys
+import os
+
+
+def check_devimg(env):
+    """
+    return the image repo/tag for devimg
+    prefer the image for this environment if one exists; otherwise the latest
+    exit the program with a warning if no image is available
+    """
+    for tag in (env.name, 'latest'):
+        devimg = 'zendev/devimg:' + tag
+        if subprocess.check_output(['docker', 'images', '-q', devimg]) != '':
+            return devimg
+    print >> sys.stderr, ("You don't have the devimg built. Please run"
+                          " zendev devimg\" first.")
+    sys.exit(1)
+
+def get_mounts(env):
+    """
+    return a map of mount points in the form of
+    OS-local-directory-name:container-local-directory-name
+    """
+    envvars = env.envvars()
+    envvars['HOME'] = os.getenv('HOME')
+    print "envvars=%s" % envvars
+    mounts = {
```

Comment:
Any way we can share the list of mounts into devimg https://github.com/zenoss/zendev/blob/zendev2/zendev/cmd/serviced.py#L370